<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmAbrirCerrarCaja</class>
 <widget class="QDialog" name="frmAbrirCerrarCaja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Abrir / cerrar caja</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources.qrc">
    <normaloff>:/assets/marpajaros.ico</normaloff>:/assets/marpajaros.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="lblInstrucciones">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Introduzca el importe en caja. Puede introducir el importe indicando el número total de monedas y billetes de cada tipo o simplemente introduciendo el total. El número de monedas y billetes es irrelevante, simplemente le ayuda a contar más fácilmente.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="grdCash">
       <item row="2" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="lblMonedas2cent">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/2centimos.gif);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numMonedas2cent">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>0.020000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lblBilletes200">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/200euros.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numBilletes200">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>200.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblBilletes20">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/20euros.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numBilletes20">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>20.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="lblBilletes10">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/10euros.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numBilletes10">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="lblMonedas50cent">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/50centimos.gif);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numMonedas50cent">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="lblBilletes100">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/100euros.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numBilletes100">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="lblMonedas20cent">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/20centimos.gif);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numMonedas20cent">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>0.200000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="lblBilletes5">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/5euros.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numBilletes5">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblMonedas1euro">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/1euro.gif);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numMonedas1">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lblMonedas5cent">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/5centimos.gif);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numMonedas5cent">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>0.050000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="lblMonedas1cent">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/1centimo.gif);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numMonedas1cent">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblBilletes500">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/500euros.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numBilletes500">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="lblBilletes50">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/50euros.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numBilletes50">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>50.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="lblMonedas2euros">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/2euro.gif);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numMonedas2">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="lblMonedas10cent">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/assets/billetesmonedas/10centimos.gif);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numMonedas10cent">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="cashValue" stdset="0">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="1">
          <widget class="QLabel" name="lblVentas">
           <property name="text">
            <string>0 €</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="lblDescuadre">
           <property name="text">
            <string>0 €</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="lblDeberiaHaber">
           <property name="text">
            <string>0 €</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lblImporteInicial">
           <property name="text">
            <string>0 €</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>TOTAL:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblVentasLabel">
           <property name="text">
            <string>Ventas:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="CRelaxedDoubleSpinBox" name="numTotal">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>0</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="suffix">
              <string> €</string>
             </property>
             <property name="maximum">
              <double>99999999.989999994635582</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblDeberiaHaberLabel">
           <property name="text">
            <string>Importe que debería haber:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblDescuadreLabel">
           <property name="text">
            <string>Descuadre:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblImporteInicialLabel">
           <property name="text">
            <string>Importe inicial:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancelar">
         <property name="text">
          <string>&amp;Cancelar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAceptar">
         <property name="text">
          <string>&amp;Aceptar</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRelaxedDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>crelaxeddoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnAceptar</tabstop>
  <tabstop>btnCancelar</tabstop>
  <tabstop>numBilletes500</tabstop>
  <tabstop>numBilletes200</tabstop>
  <tabstop>numBilletes100</tabstop>
  <tabstop>numBilletes50</tabstop>
  <tabstop>numBilletes20</tabstop>
  <tabstop>numBilletes10</tabstop>
  <tabstop>numBilletes5</tabstop>
  <tabstop>numMonedas2</tabstop>
  <tabstop>numMonedas1</tabstop>
  <tabstop>numMonedas50cent</tabstop>
  <tabstop>numMonedas20cent</tabstop>
  <tabstop>numMonedas10cent</tabstop>
  <tabstop>numMonedas5cent</tabstop>
  <tabstop>numMonedas2cent</tabstop>
  <tabstop>numMonedas1cent</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>frmAbrirCerrarCaja</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numTotal</sender>
   <signal>valueChanged(double)</signal>
   <receiver>frmAbrirCerrarCaja</receiver>
   <slot>CalcularImportes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAceptar</sender>
   <signal>clicked()</signal>
   <receiver>frmAbrirCerrarCaja</receiver>
   <slot>SaveCuadreCaja()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CalcularImportes()</slot>
  <slot>SaveCuadreCaja()</slot>
 </slots>
</ui>
